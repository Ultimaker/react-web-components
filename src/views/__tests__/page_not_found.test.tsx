// Copyright (c) 2018 Ultimaker B.V.
import * as React from 'react';
import { shallow } from 'enzyme';

// component
import PageNotFoundView from '../page_not_found';

describe('The PageNotFoundView component', () => {
    let props;
    let wrapper;

    beforeEach(() => {
        props = {
            location: {
                path: 'home',
            },
        };
        wrapper = shallow(<PageNotFoundView {...props} />);
    });

    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
