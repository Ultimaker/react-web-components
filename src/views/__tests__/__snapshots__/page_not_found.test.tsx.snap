// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The PageNotFoundView component should render 1`] = `
<Page
  maxWidth={550}
  title="Page Not Found"
>
  <Tile
    align="center"
    appearance="shadow"
    padding="lg"
  >
    <p>
      The page requested could not be found
    </p>
    <pre />
  </Tile>
</Page>
`;
